.circleci/config.yml

```diff
@@ -997,7 +929,7 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
-  test-e2e-chrome-snaps-flask:
+  test-e2e-chrome-flask:
```

Comment:
Looks like you didn't update the command for `test-e2e-chrome-flask` and `test-e2e-firefox-flask`